lms/djangoapps/grades/tests/test_signals.py

```diff
@@ -203,7 +206,7 @@ def set_up_course(self, enable_subsection_grades=True):
     def test_subsection_grade_updated_on_signal(self, default_store):
         with self.store.default_store(default_store):
             self.set_up_course()
-            with check_mongo_calls(2) and self.assertNumQueries(15):
+            with check_mongo_calls(2) and self.assertNumQueries(19):
```

Comment:
So digging further, since `ConfigurationModel.current` caches the record once warmed up, only 1 additional query would be made: to check for the course-specific setting.